packages/components/modules/messages/ChatRoomsList/ChatRoomItem/index.tsx

```diff
@@ -27,7 +27,7 @@ const ChatRoomItem: FC<ChatRoomItemProps> = ({
     if (handleClick) handleClick()
   }
 
-  const { profile } = useCurrentProfile()
+  const profile = useCurrentProfile().currentProfile
```

Comment:
@tsl-ps2 it would be nice if we could follow a pattern, since we more often use the Destructuring pattern, we should stick with it, instead of directly accessing the property

Reply:
@anicioalexandre I mainly did this because I found `const { currentProfile: profile } = useCurrentProfile()` not very readable (i.e. when re-assigning a name, I did not use destructuring). But happy to switch.

Reply:
@tsl-ps2  yee I got the point, but I feel that restructuring is better and for these cases `{ currentProfile: profile }` I think we should just use `{ currentProfile }` instead and change the references